x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/optimizer/Optimizer.java

```diff
@@ -127,8 +128,8 @@ public LogicalPlan optimize(LogicalPlan verified) {
                 new CombineProjections(),
                 // folding
                 new ReplaceFoldableAttributes(),
-                new FoldNull(),
                 new ConstantFolding(),
+                new FoldNull(),
```

Comment:
Again, why was `FoldNull` moved?

Reply:
no reason, missed the initial place, will change.